omnigibson/object_states/particle_modifier.py

```diff
@@ -632,9 +637,28 @@ def requires_overlap(self):
     def supported_active_systems(cls):
         """
         Returns:
-            list: All systems used in this state that are active, dynamic across time
+            dict: Maps system names to corresponding systems used in this state that are active, dynamic across time
         """
-        return list(VisualParticleSystem.get_active_systems().values()) + list(PhysicalParticleSystem.get_active_systems().values())
+        return dict(**VisualParticleSystem.get_active_systems(), **PhysicalParticleSystem.get_active_systems())
```

Comment:
Since you changed the return type from list to dict, did you also change other parts of the code that calls this function accordingly?

Reply:
I think you did. feel free to resolve if you agree.